config/application.rb

```diff
@@ -12,7 +12,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('app', 'mailers', 'concerns')
     config.time_zone = 'UTC'
 
-    config.middleware.use Rack::Attack unless Figaro.env.enable_load_testing_mode == 'true'
+    config.middleware.use Rack::Attack unless Figaro.env.disable_email_sending == 'true'
```

Comment:
What if we had an explicit setting for `use_rack_attack`? According to our README we already have 3 configs to set for load testing mode, why not make it extra clear and set a 4th?